src/Symfony/Bridge/PhpUnit/bin/simple-phpunit

```diff
@@ -120,7 +120,7 @@ EOPHP
 }
 
 $components = array();
-$cmd = array_map('escapeshellarg', $argv);
+$cmd = array_map('escapeshellarg', $_SERVER['argv']);
 $exit = 0;
 
 if (isset($argv[1]) && 'symfony' === $argv[1] && !file_exists('symfony') && file_exists('src/Symfony')) {
```

Comment:
if you happen to be running this script in a context where $argv is not defined, then this line will always eval to false, which means it's useless
$argv is expected to be always defined in the current code. A better patch would be to always ensure the vars are defined

Reply:
So the `$cmd` should be `$argv` to be sure it is always defined. That means that this is correct code but leaves `$argv` still used, not sure where to init the variable `$_SERVER['argv']` so that `$argv` will be intact.

Reply:
just start the script with:
```
$argv = isset($_SERVER['argv']) ? $_SERVER['argv'] : array();
$argc = isset($_SERVER['argc']) ? $_SERVER['argc'] : 0;
```

Reply:
Ah yea. I was thinking way too deep here. Thank you for your suggestion. I reverted everything in 8fbf912 and added the lines at the beginning.